cmd/rukpakctl/cmd/content.go

```diff
@@ -28,9 +28,10 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/clientcmd"
 	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	config2 "sigs.k8s.io/controller-runtime/pkg/client/config"
```

Comment:
nit: could we rename to config? I don't see a collision with another package here.